pydra/engine/workers.py

```diff
@@ -68,8 +92,12 @@ def close(self):
 
 
 class DaskWorker(Worker):
+    @utils._check_dependencies("dask", "distributed")
```

Comment:
this might be also useful for mp and cf

Reply:
`multiprocessing` and `concurrent.futures` are part of the standard python3 library so everyone should have them

Reply:
makes sense